content/getting-started/monorepo-apps.md

```diff
@@ -0,0 +1,58 @@
+---
+description: How to white label apps with Codemagic
+title: White label apps
+weight: 11
+---
+
+Monorepo is a software development strategy that allows you to store many projects in a single repository. Codemagic supports working with monorepo apps. The very first step is creating a Codemagic account and adding your monorepo app from its codebase such as GitHub, Gitlab, Bitbucket or self-hosted repositories. More information about how to sign up can be found here.
```

Comment:
A monorepo is a version-controlled code repository that holds many projects, not sure I would refer to it as a software development strategy.